homeassistant/components/camera/canary.py

```diff
@@ -4,19 +4,30 @@
 For more details about this platform, please refer to the documentation at
 https://home-assistant.io/components/camera.canary/
 """
+import asyncio
 import logging
+from datetime import timedelta
 
-import requests
+import voluptuous as vol
 
-from homeassistant.components.camera import Camera
+from homeassistant.components.camera import Camera, PLATFORM_SCHEMA
 from homeassistant.components.canary import DATA_CANARY, DEFAULT_TIMEOUT
+from homeassistant.components.ffmpeg import DATA_FFMPEG
+from homeassistant.const import CONF_SCAN_INTERVAL
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.aiohttp_client import async_aiohttp_proxy_stream
 
-DEPENDENCIES = ['canary']
+CONF_FFMPEG_ARGUMENTS = 'ffmpeg_arguments'
+
+DEPENDENCIES = ['canary', 'ffmpeg']
 
 _LOGGER = logging.getLogger(__name__)
 
-ATTR_MOTION_START_TIME = "motion_start_time"
-ATTR_MOTION_END_TIME = "motion_end_time"
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(CONF_FFMPEG_ARGUMENTS): cv.string,
+    vol.Optional(CONF_SCAN_INTERVAL, default=timedelta(seconds=90)):
```

Comment:
Remove this and define `SCAN_INTERVAL` with the default value at the module level. This config option is already part of the base platform schema.